v03_pipeline/lib/misc/gcp.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+
+import google.auth
+import google.auth.transport.requests
+import google.oauth2.credentials
+import pytz
+
+SERVICE_ACCOUNT_CREDENTIALS = None
+SOCIAL_AUTH_GOOGLE_OAUTH2_SCOPE = [
+    'https://www.googleapis.com/auth/userinfo.profile',
+    'https://www.googleapis.com/auth/userinfo.email',
+    'openid',
+]
+ONE_MINUTE_S = 60
+
+
+def get_service_account_credentials() -> google.oauth2.credentials.Credentials:
+    global SERVICE_ACCOUNT_CREDENTIALS
```

Comment:
Why are you using the `global` keyword here?

Reply:
I think I figured it out. Is it because we don't want to reset `SERVICE_ACCOUNT_CREDENTIALS` every time `get_service_account_credentials()` is called?

Reply:
I'm changing a variable with `global` scope.  More info: https://stackoverflow.com/questions/4693120/why-isnt-the-global-keyword-needed-to-access-a-global-variable

Reply:
yeah, we just refresh the token.